index.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable array-callback-return */
```

Comment:
- Please remove all disabled linters from your codebase, remember linters help to ensure that we write code that is clean and easy to maintain. Please fix this 🙏